pkg/errors/errors.go

```diff
@@ -22,13 +22,25 @@ func NewEgressURLError(failure string) error {
 }
 
 type GenericError struct {
-	e string
+	message string
 }
 
-func (e *GenericError) Error() string { return e.e }
+func (e *GenericError) Error() string { return e.message }
 
-func NewGenericError(err error) error {
+func NewGenericError(message string) error {
 	return &GenericError{
-		e: fmt.Sprintf("generic(unhandled) error: %s ", err.Error()),
+		message: fmt.Sprintf("network verifier error: %s", message),
```

Comment:
are we ok to start each one of the generic error with `network verifier error:`? Would we want to include _generic_ type in the message e.g `unhandled error` ?
To me `network verifier error` looks a bit odd 
1- because we know that error will be always thrown from `network verifier` in verifier execution context.
2- network verifier error sounds like it can be anything like egressurl error/generic error/ any other error that's defined

Reply:
_Would we want to include generic type in the message:_ Is "generic network verifier error" better?

_we know that error will be always thrown from network verifier in verifier execution context._  : Imo, including "network verifier" name in the error message gives a better context when it's integrated with other tools and produces output in the common stdout with other tools. 

_network verifier error sounds like it can be anything like egressurl error/generic error/ any other error that's defined_ I'm not sure I understand. EgressURL error will still show as EgressURL error.. it's only when we want generic error we can use this. Did I implement it in a place it should be more specific error?

Reply:
Is "generic network verifier error" better? IMO, no :)) I think we can remove the generic one if we ok with network-verifier-error